opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -2149,6 +2153,30 @@ namespace {
     }
 }
 
+    void Schedule::init_completed_cells_lgr(const EclipseGrid& ecl_grid)
+    { 
+        if (ecl_grid.is_lgr())
+        {
+            std::size_t num_label = ecl_grid.get_all_lgr_labels().size();
+            completed_cells_lgr.reserve(num_label);
+            for (auto lgr_tag : ecl_grid.get_all_lgr_labels())
+            {
+                const auto lgr_grid = ecl_grid.getLGRCell(lgr_tag);    
```

Comment:
This makes `lgr_grid` an independent copy of the `EclipseGridLGR` object returned from `getLGRCell()`.  Please use a reference to `const` instead.